docs/installation/automatic-data-balancing.rst

```diff
@@ -219,23 +141,17 @@ This tutorial runs Confluent Auto Data Balancing (ADB) on Kafka, which allows yo
 
    .. codewithvars:: bash
 
-    docker run \
-      --net=host \
-      --rm \
-      confluentinc/cp-enterprise-kafka:|release| \
-      bash -c "confluent-rebalancer status --zookeeper localhost:22181"
+    docker-compose exec broker \
+    bash -c "confluent-rebalancer status --zookeeper localhost:22181"
```

Comment:
I think this should fail because the exec is on `broker` container, but the zookeeper connection is `localhost` (which is wrong).  It should be something like `--zookeeper <name of zookeeper container>:22181"`